src/Umbraco.Core/Extensions/ObjectExtensions.cs

```diff
@@ -532,6 +535,7 @@ internal static void CheckThrowObjectDisposed(this IDisposable disposable, bool
     /// <param name="o"></param>
     /// <param name="ignoreProperties">Properties to ignore</param>
     /// <returns></returns>
+    [Obsolete("Use of this can be replaced with RouteValueDictionary or HtmlHelper.AnonymousObjectToHtmlAttributes(). The method will be removed in Umbraco 16.")]
```

Comment:
```suggestion
    [Obsolete("Use of this can be replaced with RouteValueDictionary or HtmlHelper.AnonymousObjectToHtmlAttributes(). The method will be removed in Umbraco 17.")]
```